generated package.json is missing a 'version' attribute

because of this `npm install` fails with the following error message.

```
npm ERR! install Couldn't read dependencies
npm ERR! Error: no version
npm ERR!     at validVersion (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:527:32)
npm ERR!     at final (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:289:23)
npm ERR!     at /usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:119:33
npm ERR!     at cb (/usr/local/lib/node_modules/npm/node_modules/slide/lib/async-map.js:48:11)
npm ERR!     at /usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:233:59
npm ERR!     at Glob.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/glob/glob.js:91:7)
npm ERR!     at Glob.EventEmitter.emit (events.js:88:17)
npm ERR!     at Glob._processEmitQueue (/usr/local/lib/node_modules/npm/node_modules/glob/glob.js:270:10)
npm ERR!     at Glob.emitMatch (/usr/local/lib/node_modules/npm/node_modules/glob/glob.js:255:8)
npm ERR!     at Glob._finish (/usr/local/lib/node_modules/npm/node_modules/glob/glob.js:217:8)
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Darwin 11.4.2
npm ERR! command "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! cwd /private/tmp/test
npm ERR! node -v v0.9.2
npm ERR! npm -v 1.1.61
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /private/tmp/test/npm-debug.log
npm ERR! not ok code 0
```

When I add a `version` attribute `npm install` runs without problems.

--- a/app/index.js
+++ b/app/index.js
@@ -36,6 +36,10 @@ RevealGenerator.prototype.askFor = function askFor() {
   var prompts = [{
     name: 'presentationTitle',
     message: "What are you going to talk about?"
+  },{
+    name: 'packageVersion',
+    message: "What version should we put in the package.json file?",
+    default: '0.0.0'
   }];
 
   this.prompt(prompts, function (err, props) {
@@ -43,6 +47,7 @@ RevealGenerator.prototype.askFor = function askFor() {
       return this.emit('error', err);
     }
     this.presentationTitle = props.presentationTitle;
+    this.packageVersion = props.packageVersion
     cb();
   }.bind(this));
 };

--- a/app/templates/_package.json
+++ b/app/templates/_package.json
@@ -1,5 +1,6 @@
 {
   "name": "<%= _.slugify(presentationTitle) %>",
+  "version": "<%= packageVersion %>",
   "private": true,
   "devDependencies": {
     "grunt": "~0.4.1",

--- a/test/test-creation.js
+++ b/test/test-creation.js
@@ -23,7 +23,8 @@ describe('reveal generator', function () {
     var expected = [
       // add files you expect to exist here.
       '.jshintrc',
-      '.editorconfig'
+      '.editorconfig',
+      'package.json'
     ];
 
     helpers.mockPrompt(this.app, {
@@ -33,6 +34,20 @@ describe('reveal generator', function () {
     this.app.run({}, function () {
       helpers.assertFiles(expected);
       done();
+    });
+  });
+
+  describe('generated package.json', function () {
+    it('should have a version', function (done) {
+      helpers.mockPrompt(this.app, {
+        'presentationTitle': 'package.json test',
+        'packageVersion': '1.2.3'
+      });
+      this.app.options['skip-install'] = true;
+      this.app.run({}, function(){
+        helpers.assertFile('package.json', /"version": "1.2.3"/);
+        done();
+      });
     });
   });
 });

